core/websock.js

```diff
@@ -173,49 +175,92 @@ export default class Websock {
         this._sQ = new Uint8Array(this._sQbufferSize);
     }
 
+    _getChannelStates(channelType) {
+        if (channelType === "WebSocket") {
+            return {
+                CONNECTING: WebSocket.CONNECTING,
+                OPEN: WebSocket.OPEN,
+                CLOSING: WebSocket.CLOSING,
+                CLOSED: WebSocket.CLOSED
+            };
+        }
+        if (channelType === "RTCDataChannel") {
+            // Constants pulled from RTCDataChannelState enum
+            // https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/readyState#RTCDataChannelState_enum
+            return {
+                CONNECTING: "connecting",
+                OPEN: "open",
+                CLOSING: "closing",
+                CLOSED: "closed"
+            };
+        }
+        throw new Error(`channelType: ${channelType} not recognized`);
```

Comment:
Can't we just check both possible values in the if clauses? I think that makes things easier to follow as you don't have to keep track of this extra mapping table.